express/express.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
```

Comment:
For you app express and app should use same package.json file: In the future, when you set up the SSR, your express app will require access to your code